src/sentry/static/sentry/app/views/events/searchBar.jsx

```diff
@@ -24,6 +24,15 @@ const SEARCH_SPECIAL_CHARS_REGEXP = new RegExp(
 const FIELD_TAGS = Object.fromEntries(
   Object.keys(FIELDS).map(item => [item, {key: item, name: item}])
 );
+function create_aggregation(item) {
+  const aggregation = `${item}(${AGGREGATIONS[item].parameters
+    .map(param => param?.defaultValue)
+    .join()})`;
```

Comment:
Won't this not include `,` between the arguments of `percentile()`? Should we use the currently selected fields as inputs to the function list? Right now it seems like a user could select apdex(800) and the autocomplete would suggest apdex(300) which isn't ideal.

Reply:
`.join()` defaults to `,` But I'll be more explicit in the future and include it.

Also that's a really good point, I'll update this to use selected columns instead :+1:

Reply:
Neat, I didn't know that Array.join() defaulted to , :+1: